SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -63,11 +63,11 @@ using namespace SpatialGDK;
 
 namespace // Anonymous namespace
 {
-struct RepNotifyCall
+struct ChannelObjectsToBeResolved
 {
 	USpatialActorChannel* Channel;
```

Comment:
Think this should work 
```suggestion
	TWeakPtr<USpatialActorChannel> Channel;
```